.gitignore

```diff
@@ -1,131 +1,23 @@
-*.suo
-*.user
-.DS_Store
-
-# Logs
-logs
-*.log
-
-# Runtime data
-pids
-*.pid
-*.seed
-
-# Dependency directory
-# https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git
-node_modules
-npm-debug.log
```

Comment:
While npm-debug.log will no longer be put into cwd [as of 13 days ago](https://github.com/npm/npm/commit/04fca223a0f704b69340c5f81b26907238fad878), I would maybe keep this :)